sandbox/src/mocks/r4/NHSDigital-OperationOutcome.json

```diff
@@ -1,24 +1,21 @@
 {
   "resourceType": "OperationOutcome",
   "meta": {
-    "profile": [
-      "https://fhir.nhs.uk/StructureDefinition/NHSDigital-OperationOutcome"
-    ],
     "lastUpdated": "2022-03-01T10:00:00.42Z"
   },
   "issue": [
     {
       "severity": "error",
-      "code": "forbidden",
+      "code": "example-issue-code",
```

Comment:
Was this change based on feedback? This isn't a valid code so wouldn't validate http://hl7.org/fhir/R4/valueset-issue-type.html

Reply:
This was me trying to address this

> Under the 401 response should the example use one of the 401 errors? Maybe we don't usually go into this detail I'm not sure?"

I tried to make the one example we have for the NHSDOperationOutcome a one-for-all. 

Given this is just an example for OAS documentation for clients having an idea of what to expect, does it need to strictly pass FHIR validation, i.e. values being on the typeset, etc?

Alternatively I can create an example for each of the expected errors without using these 'example codes'.

Reply:
Created examples for each HTTP error.